source/patterns/@aws-solutions-constructs/core/lib/apigateway-helper.ts

```diff
@@ -15,7 +15,7 @@
 import * as logs from 'aws-cdk-lib/aws-logs';
 import * as lambda from 'aws-cdk-lib/aws-lambda';
 import * as cdk from 'aws-cdk-lib';
-import * as api from 'aws-cdk-lib/aws-apigateway';
+import * as apig from 'aws-cdk-lib/aws-apigateway';
```

Comment:
Minor and don't necessarily need to change, but `api` and `apig` are still really close when looking at the code. I would prefer this import to be `apigateway` (which also matches the convention of the other imports in this file.